android/src/main/java/com/reactnativecommunity/webview/RNCWebViewManager.java

```diff
@@ -159,6 +159,7 @@ public class RNCWebViewManager extends SimpleViewManager<WebView> {
   protected boolean mAllowsFullscreenVideo = false;
   protected @Nullable String mUserAgent = null;
   protected @Nullable String mUserAgentWithApplicationName = null;
+  protected String mDownloadMessage = "Downloading";
```

Comment:
I'd suggest making "Downloading" a string constant, so for example it could still be used if a custom string was specified and then reset back to `undefined`? (though admittedly I don't know how likely that is... just seems like it would be a small change to be extra safe?)

Reply:
Created a separate variable for default strings